examples/src/main/kotlin/examples/onnx/objectdetection/efficientdet/EfficientDetD2.kt

```diff
@@ -28,8 +28,8 @@ fun main() {
         val preprocessing: Preprocessing = preprocess {
             transformImage {
                 resize {
-                    outputHeight = it.inputShape[1].toInt()
-                    outputWidth = it.inputShape[2].toInt()
+                    outputHeight = it.inputDimensions[0].toInt()
```

Comment:
In the future, we could consider supporting batch input because it's quite a popular approach to increasing throughput when users need to inference model on a full dataset